### PR TITLE
Bug 1120526 - Clean up facets a little and use post_filter

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: kuma/search/filters.py [hunk 1]
```diff
@@ -162,50 +161,36 @@ def filter_queryset(self, request, queryset, view):
...
+                    active_filters.append(F(filter_operator, tag_filters))
                 else:
                     active_filters.append(F('term', tags=filter_tags[0]))
 
             if len(filter_tags) > 1:
-                facet_params = {
-                    'or': {
-                        'filters': [{'term': {'tags': tag}}
-                                    for tag in filter_tags],
-                        '_cache': True,
```
Comment (jezdez) [inline kuma/search/filters.py]: IIRC that was important to keep speed high

File: kuma/search/views.py [hunk 2]
```diff
@@ -43,8 +42,6 @@ class SearchView(ListAPIView):
     def initial(self, request, *args, **kwargs):
         super(SearchView, self).initial(request, *args, **kwargs)
         self.current_page = self.request.QUERY_PARAMS.get(self.page_kwarg, 1)
-        self.drilldown_faceting = flag_is_active(request,
-                                                 'search_drilldown_faceting')
```
Comment (jezdez) [inline kuma/search/views.py]: :wink:

File: kuma/search/filters.py (hunk 1 above)
Comment (robhudson) [inline kuma/search/filters.py]: When I read the docs it was the default. They offer `_cache` if you want to disable it.

Comment (robhudson) [inline kuma/search/filters.py]: For reference: http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/query-dsl-term-filter.html#_caching_18

Comment (jezdez) [inline kuma/search/filters.py]: ok, that was different I'm sure

**Pull request merged**